### PR TITLE
Header

### PR DESCRIPTION
**Pull request by dandelionadia**
## What was done:
* finished setup
* adds responsive Header
*  default layout

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export const Header = () => {
+  return (
+    <div className="grid grid-cols-2  lg:grid-cols-3 items-center justify-items-center mx-auto relative text-sm py-6">
+      <div className="hidden lg:block"></div>
+      <div className="z-10 justify-self-start lg:justify-self-auto">
+        <Link to="#" className="block px-8 bg-gray-800">
+          <Logo className="w-12 h-12 text-white bg-gray-800 fill-current" />
+        </Link>
+      </div>
+      <div className="uppercase z-10 flex gap-x-5 font-medium tracking-widest text-violet-light text-opacity-60">
```
Comment (kettanaito) [inline src/components/Header.jsx]: If you want to have a semantic markup, these links should be written as navigation:

```jsx
<nav>
  <ul>
    <li><Link to="#">Work</Link></li>
    <li><Link to="#">Contacts</Link></li>
  </ul>
</nav>
```

File: src/pages/index.js [hunk 2]
```diff
@@ -1,184 +1,13 @@
...
-    color: "#663399",
-  },
-]
-
-// markup
 const IndexPage = () => {
   return (
-    <main style={pageStyles}>
+    <main className="container m-auto">
       <title>Home Page</title>
```
Comment (kettanaito) [inline src/pages/index.js]: You may want to change the title of your page to be something custom.

File: tailwind.config.js [hunk 3]
```diff
@@ -1,11 +1,44 @@
...
+    },
+    container: {
+      padding: {
+        DEFAULT: "1rem",
+        sm: "2rem",
+        lg: "4rem",
+        xl: "5rem",
+        "2xl": "6rem",
+      },
+      screens: {
```
Comment (kettanaito) [inline tailwind.config.js]: I wouldn't recommend changing the default breakpoints of Tailwind. You can use their default breakpoints like `sm:`, `lg:`, etc. without having to customize the `screens` here. You can remove the `screens` property.

File: tailwind.config.js [hunk 4]
```diff
@@ -1,11 +1,44 @@
...
+      colors: {
+        violet: {
+          light: "#C4D1FE",
+          dark: "#3C3CBE",
+          DEFAULT: "#6465F1",
+        },
+      },
+    },
+    container: {
+      padding: {
```
Comment (kettanaito) [inline tailwind.config.js]: Are you changing this because the default container padding is not what you want?

**Review by kettanaito — CHANGES_REQUESTED** [review]
Great changes! Please, I've left a few comments and would love for you to go through them. Thanks.

File: tailwind.config.js (hunk 4 above)
Comment (dandelionadia) [reply tailwind.config.js]: No. I didn't know there are default properties. I just tried to start working with the prepared setup.

File: src/components/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+      </div>
+      <nav className="z-10">
+        <ul className="uppercase flex gap-x-5 font-medium tracking-widest text-violet-light text-opacity-60">
+          <li>
+            <Link to="#work" className="px-4 bg-gray-800 block">
+              <div>my work</div>
+            </Link>
+          </li>
+          <li>
+            <Link to="#contacnts" className="px-4 bg-gray-800 block">
```
Comment (kettanaito) [inline src/components/Header.jsx]:
```suggestion
            <Link to="#contacts" className="px-4 bg-gray-800 block">
```

File: tailwind.config.js [hunk 6]
```diff
@@ -1,11 +1,20 @@
...
+    extend: {
+      colors: {
+        violet: {
+          light: "#C4D1FE",
+          dark: "#3C3CBE",
+          DEFAULT: "#6465F1",
+        },
+      },
+    },
+    container: {},
```
Comment (kettanaito) [inline tailwind.config.js]: You can remove the property if you're not changing/extending it.

**Review by kettanaito — CHANGES_REQUESTED** [review]

**Review by kettanaito — APPROVED** [review]
Wonderful! :tada:

**Pull request merged**